src/pooledembeddings.jl

```diff
@@ -0,0 +1,87 @@
+export PooledEmbeddings
+
+using ChainRulesCore: NoTangent
+using Polynomials4ML: PooledSparseProduct, AbstractPoly4MLBasis
+
+import ChainRulesCore: rrule
+import Base.Cartesian: @nexprs
+
+struct PooledEmbeddings{NB, TB <: Tuple} <: AbstractPoly4MLBasis
```

Comment:
Should it be `PooledEmbedding` or `PooledEmbeddings`? The plural refers to the multiple embeddings. The singular refers to the output of the layer, which is a *single* embedding of the multi-set.  I prefer the singular here if that's ok.